Change RPC endpoints and providers section (#326)
OffchainLabs/arbitrum-docs: arbitrum-docs/node-running/node-providers.mdx

:::

## RPC endpoints

This section provides an overview of the available public RPC endpoints for different Arbitrum chains and necessary details to interact with them.

| Name                      | RPC Url(s)                            | Chain ID | Block explorer             | Underlying L1 | Tech stack | Sequencer feed URL                   |
| ------------------------- | ------------------------------------- | -------- | -------------------------- | ------------- | ---------- | ------------------------------------ |
| Arbitrum One              | https://arb1.arbitrum.io/rpc          | 42161    | https://arbiscan.io/       | Ethereum      | Nitro      | wss://arb1.arbitrum.io/feed          |
| Arbitrum Nova             | https://nova.arbitrum.io/rpc          | 42170    | https://nova.arbiscan.io/  | Ethereum      | AnyTrust   | wss://nova.arbitrum.io/feed          |
| Arbitrum Goerli (Testnet) | https://goerli-rollup.arbitrum.io/rpc | 421613   | https://goerli.arbiscan.io | Goerli        | Nitro      | wss://goerli-rollup.arbitrum.io/feed |

> 🔉 Note: You can also find additional RPC endpoints available for Arbitrum chains in Chain Connect: [Arbitrum One](https://www.alchemy.com/chain-connect/chain/arbitrum-one), [Arbitrum Nova](https://www.alchemy.com/chain-connect/chain/arbitrum-nova) and [Arbitrum Goerli](https://www.alchemy.com/chain-connect/chain/arbitrum-goerli).

Reviewer: (not a high priority, but I've been replacing these `Notes` with admonitions wherever I catch them)

```suggestion

:::info More RPC endpoints

More Arbitrum chain RPC endpoints can be found in Chain Connect: [Arbitrum One](https://www.alchemy.com/chain-connect/chain/arbitrum-one), [Arbitrum Nova](https://www.alchemy.com/chain-connect/chain/arbitrum-nova) and [Arbitrum Goerli](https://www.alchemy.com/chain-connect/chain/arbitrum-goerli).

:::

```
